apps/coordinator/src/components/BCUR2/BCUR2Demo.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Container,
+  Typography,
+  Paper,
+  Tab,
+  Tabs,
+  Grid,
+} from "@mui/material";
+import { BCUR2Display, BCUR2PSBTExporter } from "./index";
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`bcur2-tabpanel-${index}`}
+      aria-labelledby={`bcur2-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
+    </div>
+  );
+}
+
+const BCUR2Demo: React.FC = () => {
+  const [tabValue, setTabValue] = useState(0);
+
+  const handleTabChange = (_event: React.SyntheticEvent, newValue: number) => {
+    setTabValue(newValue);
+  };
+
+  // Sample PSBT for demonstration
+  const samplePSBT = "cHNidP8BAHsCAAAAAljoeiG1ba8MI76OcHBFbDNvfLqlyHV5JPVFiHuyq911AAAAAAD/////g40EJ9DsZQpoqka7CwmK6kQiwHGyyng1Kgd5WdB86h0BAAAAAP////8CcKrwCAAAAAAWABTYXCtx0AYLCcmIauuBXlCZHdoSTQDh9QUAAAAAFgAUAK6pouXw+HaliN9VRuh0LR2HAI8AAAAAAAABAP1pAQIAAAABAbMN9k5PaVBvfEFrI1/5/FfRJLh5pvfqpN/8tF5x4GgAAAAAAAAAAAAA/////wM8GgGJAAAAAAGpY3ELs5+bIQJYAhEUhI0iAAAIVWEhAk9R0CCoAA7gvgAA5o8OUGSyiJv7tRg3mHOwjqgfhzuH3CfzCABkVwNqFAZGQAA0AFdIYUOKhFoaKaIHewLY+hH7gGkZJr1NG5FJZJc8F2hIlv8AAADq0tAAVUHy7Wk5IqJmOJKsAAAQS6LGU51vqYIIQlwBAAAACwABDwABEAICAD/////LAQEJAQNvPKESTwAAAAj6/////wERCQEEWqkJjAAAAtX/////AREJAQIoAAACAAAAAP//";
```

Comment:
Seems this is an invalid PSBT? I tried to inspect it in bip370.org. Maybe that's expected.